src/app/qan/qan-table/qan-table.component.ts

```diff
@@ -0,0 +1,107 @@
+import {Component, OnChanges, OnDestroy, OnInit, ViewChild} from '@angular/core';
+import {PerfectScrollbarComponent, PerfectScrollbarConfigInterface} from 'ngx-perfect-scrollbar';
+import {QueryParams} from '../../core/core.component';
+import {SelectOptionModel} from '../qan-table-header-cell/modesl/select-option.model';
+import {TableDataModel} from './models/table-data.model';
+import {MetricModel} from './models/metric.model';
+import {ProfileService} from '../../inventory-api/services/profile.service';
+import {Subscription} from 'rxjs/internal/Subscription';
+import {filter, map} from 'rxjs/operators';
+import {MetricsNamesService} from '../../inventory-api/services/metrics-names.service';
+import {GetProfileBody, QanTableService} from './qan-table.service';
+import {ParseQueryParamDatePipe} from '../../shared/parse-query-param-date.pipe';
+import {ActivatedRoute, Router} from '@angular/router';
+
+@Component({
+  selector: 'app-qan-table',
+  templateUrl: './qan-table.component.html',
+  styleUrls: ['./qan-table.component.scss']
+})
+export class QanTableComponent implements OnInit, OnDestroy {
+  @ViewChild(PerfectScrollbarComponent) componentRef?: PerfectScrollbarComponent;
+
+  public scrollbarConfig: PerfectScrollbarConfigInterface = {
+    suppressScrollY: false
+  };
+
+  public iframeQueryParams: QueryParams;
+  public measurement: string;
+  public queryProfile: Array<{}>;
+  public tableData: TableDataModel[];
+  public totalRows: number;
+  public queries: any;
+  public report$: Subscription;
+  public metrics$;
+  public metrics;
+  private parseQueryParamDatePipe = new ParseQueryParamDatePipe();
+  public getReportParams = {} as GetProfileBody;
+  public prevGetReportParams = {} as GetProfileBody;
+  public groupBy: string;
+
+  public from;
+  public to;
+  private routerSubscription$: Subscription;
+
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router,
+    private profileService: ProfileService,
+    private metricsNamesService: MetricsNamesService,
+    private qanTableService: QanTableService
+  ) {
+    this.iframeQueryParams = this.route.snapshot.queryParams as QueryParams;
+    this.from = this.parseQueryParamDatePipe.transform(this.iframeQueryParams.from, 'from');
+    this.to = this.parseQueryParamDatePipe.transform(this.iframeQueryParams.to, 'to');
+
+    this.metrics$ = this.metricsNamesService.GetMetricsNames({})
+      .pipe(map(metrics => metrics.data))
+      .subscribe(metrics =>
+        this.metrics = Object.entries(metrics).map(metric => new SelectOptionModel(metric)));
+
+    this.report$ = this.profileService.GetReport({
+      'period_start_from': this.from.utc().format('YYYY-MM-DD HH:mm:ss'),
```

Comment:
Looks like we need to change format: 'YYYY-MM-DD HH:mm:ss' -> 'YYYY-MM-DDTHH:mm:ssZ'